### PR TITLE
[TECHNICAL SUPPORT] LPS-172274 Delete & Expire button - Not working as expected

### PR DESCRIPTION
**Pull request by NemethNorbert**
Previous PR: https://github.com/liferay-frontend/liferay-portal/pull/2975

Changes: Instead of changing the behaviour I made sure we recalculate the actionItems and their disabled state on first render, when there is at least one selection.

Please review my changes

/cc @markocikos

### CODE REVIEW

File: modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/SelectionControls.js [hunk 1]
```diff
@@ -127,13 +128,32 @@ const SelectionControls = ({
...
 
+			const allSelectedElements = select.getAllSelectedElements();
+
+			const elements = {
+				allSelectedElements,
+				currentPageElements: select._getCurrentPageElements(),
+				currentPageSelectedElements: select.getCurrentPageSelectedElements(),
+			};
+
+			if (isFirstRenderRef.current && allSelectedElements.size()) {
```
Comment (markocikos) [inline modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/SelectionControls.js:139]: Do we need this first render check? This code is in `useEffect` with no arguments and in `componentReady`, so all of it should already be executed only on first render.

**Review by markocikos — COMMENTED** [review]
@NemethNorbert Looks good! I have only one minor doubt, see inline comment.

File: modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/SelectionControls.js (hunk 1 above)
Comment (NemethNorbert) [reply modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/SelectionControls.js]: Quite right, I removed that part and force pushed the change.

Thanks,

**Review by markocikos — APPROVED** [review]
LGTM! :sparkles:

**Comment by markocikos** [comment]
ci:forward

**Comment by markocikos** [comment]
ci:forward

**Pull request closed without merging**